### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by Esmatullah12**
Add form to HTML file and give exact style according to the Figma template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,6 +203,23 @@ <h1 class="about-title">About Myself</h1>
...
+                <div class="contact-info">
+                    <h2>Contact me</h2>
+                    <p>if you have an application you are intersted in developing, a feature you that you need built or a project that needs coding. I'd love to help with it!</p>
+                </div>
+                <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
+                    <input class="form-style" type="text" name="name" id="name" placeholder="Enter your name" maxlength="30" required>
+                    <input class="form-style" type="email" name="email" id="email" placeholder="Enter your email address" required>
+                    <textarea class="form-style" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                    <div>
+                        <input class="regular-btn get-in-touch" type="submit" name="submit" id="submit" value="Get In Touch">
```
Comment (NahnahAJ) [inline index.html:217]: - [ ] Great job so far @Esmatullah12. Your project looks great. However, I noticed that your `Submit` button is missing the button interaction styles defined for your template. So kindly include this as well to improve user interactivity with your project :+1: 
- Button styles
image

File: index.html [hunk 2]
```diff
@@ -203,6 +203,23 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </div>
         </section>
+        <section class="contact-secion">
+            <div class="contact">
+                <div class="contact-info">
+                    <h2>Contact me</h2>
+                    <p>if you have an application you are intersted in developing, a feature you that you need built or a project that needs coding. I'd love to help with it!</p>
+                </div>
+                <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
```
Comment (NahnahAJ) [inline index.html:212]: - [ ] I also tried to submit my information through your form but it didn't send successfully. This was the feedback message below. Kindly fix this so visitors can successfully reach out to you when they fill out the form and submit it :+1: 
- Feedback 
Screenshot from 2023-03-28 17-57-09

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Esmatullah12 and @mohametalmeari , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No linter errors :heavy_check_mark: 
- Nice Design :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Required Changes ♻️
- [ ] I noticed that link to your license file in the README leads to an error page. This might be because you don't have a license file in your project. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 
Screenshot from 2023-03-28 17-50-04

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -144,4 +144,4 @@ I would like to thank Microverse for creating such wonderful opportunity for tho
 
 This project is [MIT]() licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="https://github.com/Esmatullah12/portfolio/blob/main/LICENSE.md">back to top</a>)</p>
```
Comment (jerryowusu) [inline README.md:147]: The link to your license page still leads to an error. This is due to the fact that you have not provided the reference of the license file to the link on line 145. 
Just as you provided a link for your linkedIn page 

`[LinkedIn](https://www.linkedin.com/in/esmatullah-qadeemi-b9951821b/)`, 

you should do similar. eg: `[MIT](./LICENSE`)  or `[MIT](./license)`

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost there

 ## Highlights
Form working fine :heavy_check_mark: 
No linter errors :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] Kindly consider removing the live demo section from your readme file since it is not needed now. You  can add it later when your app is hosted online. :wink: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @Esmatullah12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Formspree service integrated successfully :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**